docs/robots/husky/tutorials_husky.mdx

```diff
@@ -4,7 +4,1535 @@ sidebar_position: 4
+   for your platform (32 bit - i386, 64 bit - amd64).
+
+2. Copy the image to a USB drive using `unetbootin`, `rufus`, `balena etcher`, or a similar program. For example:
+
+    ```bash
+    sudo unetbootin isofile="clearpath-universal-noetic-amd64-0.4.17.iso"
+    ```
+
+3. Connect your robot PC to internet access (via wired Ethernet), a keyboard, and a monitor. Make sure that the
+   PC is connected to shore power or that the Husky battery is fully charged.
+
+    :::warning
+    The next step wipe your robot's hard drive, so make sure you have that image
+    [backed up](#performing-a-backup).
+    :::
+
+4. Boot your robot PC from the USB drive and let the installer work its magic.  If asked for a partitioning method
+   choose `Guided - use entire disk and set up LVM`.
+5. The setup process will be automated and may take a long time depending on the speed of your internet connection.
+6. Once the setup process is complete, the PC will turn off. Please unplug the USB drive and turn the PC back on.
+7. On first boot, the username will be `administrator` and the password will be `clearpath`. You should use the
+   `passwd` utility to change the `administrator` account password.
+8. To set up a factory-standard Husky robot, ensure all your peripherals are plugged in, and run the following command:
+
+    ```bash
+    rosrun husky_bringup install
+    ```
+
+    The install script will configure a ros upstart service, that will bring up the base Husky launch files on boot.
+    The script will also detect any standard peripherals (IMU, GPS, etc.) you have installed and add them to the service.
+
+#### Testing Base Configuration
+
+1. To test your configuration, start the background service with the following command:
+
+    ```bash
+    sudo systemctl start ros
+    ```
+
+2. The `COMM` light on your Husky should go from red to green. You can check that the service has started correctly
+   by checking the logs:
+
+    ```bash
+    sudo journalctl -u ros
+    ```
+
+3. Your Husky should now be accepting commands from your joystick. The service will automatically start each time you
+   boot your Husky's PC.
+
+#### Pairing the Controller    {#pairing-the-controller}
+
+##### PS4 Controller
+
+<ComponentPs4ControllerPairing />
+
+##### Logitech F710 Controller
+
+Some Husky robots ship with a Logitech F710 controller instead of a PS4 controller.  Pairing these controllers
+is very easy: simply plug the USB dongle into one of robot's USB ports and turn the controller on.
+
+By default Husky will use the PS4 controller for teleoperation and ignore the F710.  To enable the F710 to control
+the robot, run `sudo nano /etc/ros/setup.bash` and add the following line to the middle of the file, under the six
+`#` characters:
+
+```bash
+######
+export HUSKY_LOGITECH=1
+```
+
+Save the file and quit nano.  Then restart ROS by running `sudo systemctl restart ros` or rebooting the robot.
```

Comment:
Should we explain how to quit nano 'press CTRL + X to save and quit nano'. It sounds trivial, but I similarly struggle with instructions that use Vim without explaining the hotkeys.

Reply:
Updated